### ISSUE TITLE
Icon files/folder not syncing 

### ISSUE THREAD

**Issue by Alialsufayan** [FlorianWoelki/obsidian-iconize#551]
I believe my obsidian setup is a mess, and I'm the only on having this bug, but if you could help me 
find way to solve it i will appreciate since has been a rabbit hole for me.

The files/folder icon are not syncing but the icons inside the notes are synced and work just fine..
the weird part is ./obsidian/icons folder is empty.

**Comment by ahydronous** [comment]
I am having the same issue with Iconize + Remotely Save.

Perhaps its the old: https://github.com/FlorianWoelki/obsidian-iconize/issues/202 bug?

@FlorianWoelki

**Comment by FlorianWoelki** [comment]
I don't think this is an issue with Iconize. The issue might be related to the general syncing issues with a lot of icons.

**Comment by highnze** [comment]
I still have the same issue...

**Comment by Kribensisis** [comment]
I’ve had an issue with folder icons not syncing since the beginning with using this plugin. It’s getting to the point where I might not be able to use this anymore since it’s not practical to change the icons on 100 folders on three devices. Not sure if this is the same issue as the original user is experiencing, but I think it is. In my case, all the recommended options are enabled, such as using the right directory and enabling background sync, etc.

**Comment by Darkzarich** [comment]
I had the same issue and in the end moved from obsidian-iconize to [iconic](https://github.com/gfxholo/iconic). 
It works much better with sync. 
Hope it helps!

**Comment by Kribensisis** [comment]
@Darkzarich thanks for the suggestion! Iconize is a great extension, but reading the many mentions of this issue and seeing their age, it seems doubtful this will ever be fixed. 

I ended up starting from scratch using emojis and the default icons, which took quite some time. But then, even those didn't sync from iOS to Mac. It's not practical to use Iconize for folder icons, so I'm sure I'll just stop using this one.

**Comment by xuanyue1024** [comment]
If you're using non-custom icons (local icons), their data path is in `.obsidian\plugins\obsidian-icon-folder\data.json`. This means that if you need to sync your icon settings, you should sync this file. Considering that you're using remotely saved data, you might need to enable the "Sync configuration folder" option. This comes with some risks as it will sync all plugin configurations, though there might be better solutions. I hope this helps.

**Comment by Kribensisis** [comment]
@xuanyue1024 I actually don’t see an option with that name in the user interface.

**Comment by xuanyue1024** [comment]
@Kribensisis Got it. It might be called "Sync Config Dir (experimental)," from [https://github.com/remotely-save/remotely-save](https://github.com/remotely-save/remotely-save).

**Comment by Kribensisis** [comment]
It seems like you might mean this is a third-party plugin I’d need to install?

**Comment by xuanyue1024** [comment]
> It seems like you might mean this is a third-party plugin I’d need to install?看起来你可能是指这是一个我需要安装的第三方插件？

Yes, I'm not using the official sync feature. Perhaps the official sync library settings include syncing plugins, themes, etc. (i.e., syncing the .obsidian folder). Maybe the official documentation can help you.

**Comment by Kribensisis** [comment]
Got it. I just switched plugins since Iconize doesn’t support this use case well. The third-party plugin you mentioned should be an option _in Iconize_.

**Comment by ahydronous** [comment]
@FlorianWoelki I have experimental config dir syncing enabled and it does not sync icons

**Comment by maelp** [comment]
I have the same issue, remotely-save does not sync the iconize icons?

**Comment by SignFinder** [comment]
Unfortunately I have the same issue - Iconize with Remotely-save does not sync icons
